README.md

```diff
@@ -32,9 +32,7 @@ impl Window {
 }
 ```
 
-Without Gladis, you would have to manually parse each of the Glade entries like
-described in the [official Gtk-rs Glade
-tutorial](https://gtk-rs.org/docs-src/tutorial/glade):
+Without Gladis, you would have to manually parse each of the Glade entries.
```

Comment:
(couldn't find any updated docs for Glade usage on the gtk-rs website :/ there's always https://gtk-rs.org/gtk3-rs/stable/latest/docs/gtk/struct.Builder.html but it references C functions...)

Reply:
Fine.